ddtrace/llmobs/_experiment.py

```diff
@@ -409,8 +409,12 @@ def _run_task(self, jobs: int, raise_errors: bool = False, sample_size: Optional
                 task_results.append(result)
                 err_dict = result.get("error") or {}
                 err_msg = err_dict.get("message") if isinstance(err_dict, dict) else None
```

Comment:
nit but prob worth just doing a

```python
if isinstance(err_dict, dict):
    ...
```
at this point

Reply:
Lol true